ttl/ttl.go

```diff
@@ -9,22 +9,36 @@ import (
 	"errors"
 	"sync"
 	"time"
+
+	"github.com/juju/loggo"
+)
+
+var (
+	logLevel = "INFO"
+	log      = loggo.GetLogger("ttl")
```

Comment:
This variable is unused, which makes the `init` function and `logLevel` variable redundant.

Reply:
linter can bite me. I keep adding and removing this same block when debugging. So let's leave it in place but remain silent by default.